util/EFSOI_Utilities/src/scatter_chunks_efsoi.f90

```diff
@@ -0,0 +1,156 @@
+module scatter_chunks_efsoi
+
+use mpisetup !, only: numproc, nproc, mpi_real4
```

Comment:
@AndrewEichmann-NOAA  Another instance of blanket "use" statements that is undesirable

Reply:
fixed:
-use mpisetup !, only: numproc, nproc, mpi_real4
-use kinds, only: i_kind, r_single
+use mpisetup, only: numproc, nproc, mpi_real4
+use mpimod, only : mpi_comm_world
+use kinds, only: i_kind, r_single, r_kind